docs/developers/index.md

```diff
@@ -4,47 +4,68 @@ This section of the DataHub documentation is for developers. Here you can learn
 
 [![Gitter](https://img.shields.io/gitter/room/frictionlessdata/chat.svg)](https://gitter.im/frictionlessdata/chat)
 
-We use following repositories on GitHub and GitLab for our platform:
+We use following repositories on GitHub for our DataHub platform:
 
-* [DPR API][dpr-api] - API and web-application
-* [DPR DEPLOY][dpr-deploy] - Automated deployment of application
-* [DPR JS][dpr-js] - Visualizations and front-end JS
-* [DPR DOCS][dpr-docs] - Documentations
+* [DEPLOY][deploy] - Automated deployment
+* [FRONTEND][frontend] - Frontend application in node.js
+* [ASSEMBLER][assembler] - Data assembly line
+* [AUTH][auth] - A generic OAuth2 authentication service and user permission manager.
+* [SPECSTORE][specstore] - API server for managing a Source Spec Registry
+* [BITSTORE][bitstore] - A microservice for storing blobs i.e. files.
+* [DOCS][docs] - Documentations
+
+[deploy]: https://github.com/datahq/deploy
+[frontend]: https://github.com/datahq/frontend
+[assembler]: https://github.com/datahq/assembler
+[auth]: https://github.com/datahq/auth
+[specstore]: https://github.com/datahq/specstore
+[bitstore]: https://github.com/datahq/bitstore
+[docs]: https://github.com/datahq/docs
 
-[dpr-api]: https://github.com/frictionlessdata/dpr-api
-[dpr-deploy]: https://gitlab.com/datopian/datahub-deploy
-[dpr-js]: https://github.com/frictionlessdata/dpr-js
-[dpr-docs]: https://github.com/frictionlessdata/dpr-docs
 
 <div class="mermaid">
 graph LR
 
 subgraph Repos
-  dprapi[DPR API]
-  dprjs[DPR JS]
-  dprdeploy[DPR Deploy]
-  dprdocs[DPR Docs]
-  dprjs --submodule--> dprapi
+  frontend[Frontend]
+  assembler[Assembler]
+  auth[Auth]
+  specstore[Specstore]
+  bitstore[Bitstore]
+  docs[Docs]
+  cli[CLI]
 end
 
 subgraph Sites
-  dhio[xxx.dh.io]
-  dhdocs[docs.dh.io]
-  dprdocs --> dhdocs
+  dhio[datahub.io]
+  dhdocs[docs.datahub.io]
+  cli --> npm
+  docs --> dhdocs
 end
 
-deploy((Deploy))
-dprapi --> deploy
+deploy((DEPLOY))
 deploy --> dhio
-dprdeploy --> deploy
+frontend --> deploy
+assembler --> deploy
+auth --> deploy
+specstore --> deploy
+bitstore --> deploy
+
 </div>
 
 ## Install
 
-To install the platform locally for development purposes, please follow the instructions here:
+We use several different services to run our platform, please follow the installation instructions here:
+
+* [Install Assembler](https://github.com/datahq/assembler/blob/master/README.md)
+
+* [Install Auth](https://github.com/datahq/auth/blob/master/README.md)
 
-https://github.com/frictionlessdata/dpr-api
+* [Install Specstore](https://github.com/datahq/specstore/blob/master/README.md)
```

Comment:
I would change this link with links like that 
https://github.com/datahq/specstore#datahq-spec-store

So instead of directly to README. main repo #datahq...